src/js/media-container.js

```diff
@@ -120,25 +124,17 @@ template.innerHTML = `
     }
   </style>
 
-  <span part="layer media-layer">
-    <slot name="media"></slot>
-  </span>
-  <span part="layer poster-layer">
-    <slot name="poster"></slot>
-  </span>
-  <span part="layer gesture-layer">
-    <slot name="gestures-chrome">
-      <media-gesture-receiver slot="gestures-chrome"></media-gesture-receiver>
-    </slot>
-  </span>
+  <slot name="media" part="layer media-layer"></slot>
+  <slot name="poster" part="layer poster-layer"></slot>
+  <slot name="gestures-chrome" part="layer gesture-layer">
+    <media-gesture-receiver slot="gestures-chrome"></media-gesture-receiver>
+  </slot>
   <span part="layer vertical-layer">
-    <slot name="top-chrome"></slot>
-    <span class="spacer"><slot name="middle-chrome"></slot></span>
+    <slot name="top-chrome" part="top chrome"></slot>
+    <slot name="middle-chrome" part="middle chrome"></slot>
+    <slot name="centered-chrome" part="layer centered-layer center centered chrome"></slot>
```

Comment:
question(blocking): any concerns over nesting the layers here?

Reply:
initially it doesn't seem an issue. 
it could become an issue if a user would like the vertical layer to be a different size than 100% dimensions maybe or starting with backgrounds and opacity etc.

@heff @cjpillsbury was also wondering. was there a clear use for the middle-chrome slot? 

we could possibly rename the centered-layer to something more specific / local for this case and remove `layer centered-layer`. something like `part="center centered chrome overlay"`

Reply:
I have some concerns with this DOM restructure, but since this is mostly advanced usage/"under the hood," I don't think these concerns should block the PR. 
tl;dr - centered-chrome is intended to be used as an independent layer/layout from the vertical layer/slots, so things may get weird around the edges for advanced use cases and potential future theme use cases. If those come up, we can always revist.